src/avfoundation.cs

```diff
@@ -12300,7 +12300,7 @@ interface AVPlayerItem : NSCopying {
 		[Export ("externalMetadata", ArgumentSemantic.Copy)]
 		AVMetadataItem[] ExternalMetadata { get; set; }
 
-		[NoiOS][NoMac][NoWatch]
+		[iOS (16,0), NoMacCatalyst][NoMac][NoWatch]
```

Comment:
Lets be consistent (although I prefer your way).
```suggestion
		[iOS (16,0)][NoMacCatalyst][NoMac][NoWatch]
```